code/game/objects/items/airbubble.dm

```diff
@@ -333,6 +344,29 @@
 	else
 		to_chat(usr, "<span class='warning'>[src] has no tank.</span>")
 
+// Remove tank from bubble
+/obj/structure/closet/airbubble/verb/take_cell()
+	set src in oview(1)
+	set category = "Object"
+	set name = "Detach Power Cell"
```

Comment:
Unguarded verb. Literally any ghost or mouse can use this. Use the `CanUse()` proc to validate the user.